atlas/Snakefile

```diff
@@ -159,8 +158,14 @@ elif config.get("workflow", "complete") == "complete":
     REPORTS = ["reports/assembly_report.html"]
 
     if config.get("perform_genome_binning", True):
-        REPORTS.append("reports/bin_report.html")
-
+        # later update to include others as well as integrate concoct and metabat
+        REPORTS.append("reports/bin_report_maxbin.html")
+        # include this?
```

Comment:
@brwnj 

I think it doesn't make senes to run all binners, we cannot include the results in the final table. 

second, 
I would prefer to keep things separated, qc, assembly, gene_annotation, binning. I had a hard time to assemble my samples, and would prefer if `atlas assemble` only produces the assembly report, as a checkpoint so to say. 

If you want, we can still generate a workflow=complete. 
in this case I would include 

```
expand("reports/bin_report_{binner}.html",
        binner=config['binner'])
```